packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -36,42 +35,43 @@ export const testMessagesToSamePartitions: SendableMessageInfo[] = [
 ];
 
 export const testSessionId = "my-session";
+export const testSessionId2 = "my-session2";
```

Comment:
Since we now have `testSessionId2`, lets rename `testSessionId` to `testSessionId1`